.github/workflows/test_pytorch_wheels.yml

```diff
@@ -57,6 +57,13 @@ jobs:
   test_wheels:
     name: Test PyTorch Wheels | ${{ inputs.amdgpu_family }}
     runs-on: ${{ inputs.test_runs_on }}
+    container:
+      image: ghcr.io/rocm/no_rocm_image_ubuntu24_04:main
+      options: --ipc host
+        --group-add video
+        --device /dev/kfd
+        --device /dev/dri
+        --group-add 992
```

Comment:
This runs on Windows too, so the container should be optional here as well. We might even want to perform most steps outside of the container and then just run the actual test step (`bash external-builds/pytorch/run_linux_pytorch_tests.sh`) under docker directly (instead of using this job-level `container: ` feature)